CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,32 @@
+# v1.9.0 - Changelog since v1.8.2
+
+## Changes by Kind
+
+### Feature
+
+- Adding auto-stamped details like PV, PVC and namespace name from PV description as labels for the PVs/PDs ([#1090](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1090), [@Sneha-at](https://github.com/Sneha-at))
+- Support setting snapshot labels to Images ([#1066](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1066), [@jenting](https://github.com/jenting))
+- Pass in ProvisionedIOPSOnCreate as a parameter for CreateVolume ([#1079](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1079), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Add provisionedThroughput for hyperdisk ([#1101](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1101), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Change iops params directly convert string to int64 ([#1128](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1128), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+
+### Bug or Regression
+
+- Add udevadm binary in the container image. ([#1072](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1072), [@jenting](https://github.com/jenting))
+- Logging for gRPC call is now limited to 10000 char by default, and configurable with flag --grpc-log-char-cap ([#1085](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/1085), [@leiyiz](https://github.com/leiyiz))
```

Comment:
@hsadoyan can you add this to 1.8.2 release note? seem like it's missing from the 1.8.2 release note

Reply:
At the same time I will remove this line.

Reply:
I don't think pull/1085 was cherrypicked into 1.8.2, was it?

Reply:
it was. people who wants things to be cherrypicked should cherry pick PRs, which was me. #1111 cherry picked it